src/TemplateRenderer.php

```diff
@@ -147,34 +152,59 @@ public function renderSimpleDatasetSiteFromDataDownloads(
             "datePublished" => $settings["dateFirstPublished"],
         ]);
 
+        if (isset($settings["openBookingAPIBaseUrl"])) {
+            $dataset->setAccessService(new WebAPI([
+                "name" => "Open Booking API",
+                "description" => "An API that allows for seamless booking experiences to be created for ".
+                    strtolower($dataFeedHumanisedList)." available from ".
+                    $settings["organisationName"],
+                "authenticationAuthority" => $settings["openBookingAPIAuthenticationAuthorityUrl"],
+                "conformsTo" => array(
+                  "https://openactive.io/open-booking-api/EditorsDraft/"
+                ),
+                "documentation" => $settings["openBookingAPIDocumentationUrl"],
+                "endpointDescription" => "https://www.openactive.io/open-booking-api/EditorsDraft/swagger.json",
+                "endpointUrl" => $settings["openBookingAPIBaseUrl"],
+                "landingPage" => $settings["openBookingAPIRegistrationUrl"],
+                "termsOfService" => $settings["openBookingAPITermsOfServiceUrl"],
+            ]));
+        }
+        
+        // Also support setting accessService as a legacy feature, which is not documented
         if (isset($settings["accessService"])) {
             $dataset->setAccessService($settings["accessService"]);
         }
 
         // Only set BookingService if valid platformName provided
-        if(
+        if((
             array_key_exists("platformName", $settings) &&
             is_string($settings["platformName"]) &&
             strlen($settings["platformName"]) > 0
```

Comment:
You can replace all three of these checks with `!empty($settings["platformName"])`

Reply:
🙌 nice!